src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+
+            addRowLabel(F("<strong>Low</strong>"));
+            addFormFloatNumberBox(F("Ref Ph"),
+                                    F("Plugin_103_ref_cal_L"),
+                                    PCONFIG_FLOAT(2),
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 1) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:red;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_L"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_L\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_M\").checked = false;document.getElementById(\"Plugin_103_enable_cal_H\").checked = false;};</script>\n"));
+
+            addHtml(F("<TR><TD><strong>High</strong></TD>"));
+            addFormFloatNumberBox(F("Ref Ph"),
+                                    F("Plugin_103_ref_cal_H"),
+                                    PCONFIG_FLOAT(3),
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 2) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:orange;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_H"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_H\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;document.getElementById(\"Plugin_103_enable_cal_M\").checked = false;};</script>\n"));
+
+            if (nb_calibration_points > 1) {
+                char sensordata[32];
+                bool status;
+                status = _P103_send_I2C_command(I2Cchoice, F("Slope,?"), sensordata);
+
+                if (status) {
+                String slopeAnswer = F("Answer to 'Slope' command : ");
+                slopeAnswer += sensordata;
+                addFormNote(slopeAnswer);
+                }
+            }
+            break;
+          }
+        
+        case ORP:
+        {
+            int nb_calibration_points = -1;
+            status = _P103_send_I2C_command(I2Cchoice, F("Cal,?"), sensordata);
+
+            if (status) {
+                if (strncmp(sensordata, "?Cal,", 5)) {
```

Comment:
Can't you use 
```c++
strncmp_P(sensordata, F("?Cal,"), 5)
``` 
(or one of the other flash string marker macros)

Reply:
yes I can